aea/crypto/base.py

```diff
@@ -30,6 +30,13 @@ class Crypto(ABC):
 
     identifier = "base"
 
+    def __init__(self, **kwargs):
```

Comment:
we need this so `aea.crypto.make` can forward the keyword argument.

I can see an opportunity to apply a further refactoring. Every subclass, so far, requires the `private_key_path` input. If `None`, _generate_ a key, o/w _load_ it. These two could be two different methods, and the constructor would implement the check.

Reply:
Ok sure, can you add this?